backend/app/crud/prompt_version.py

```
@@ -0,0 +1,273 @@
+) -> PromptVersion | None:
+    """
+    Fetch a prompt version by its ID.
+    """
+    stmt = (
+        select(Prompt, PromptVersion)
+        .outerjoin(
+            PromptVersion,
+            and_(
+                Prompt.id == PromptVersion.prompt_id,
+                PromptVersion.version == version,
+                PromptVersion.is_deleted == False,
```

Ruff (0.12.2)

Avoid equality comparisons to `False`; use `not PromptVersion.is_deleted:` for false checks

Replace with `not PromptVersion.is_deleted`

(E712)